blocks/form/wizard.js

```diff
@@ -0,0 +1,48 @@
+function createButton(fd) {
+  const button = document.createElement('button');
+  button.type = 'button';
+  button.textContent = fd.Label;
+  button.className = 'form-wizard-button';
+  return button;
+}
+
+const createTooltip = (() => {
```

Comment:
small performance improvement: instead of creating a new event listener each time the function is called, we could re-use one tooltip instead:

```
let tooltip = null;

function createTooltip(fd) {
  if (!tooltip) {
    tooltip = document.createElement('div');
    tooltip.className = 'field-tooltip';
    document.addEventListener('click', () => tooltip.remove(), true); // execute for next click
  }
  tooltip.textContent = fd.Label;
  return tooltip;
}
```

Reply:
Hi @solaris007 ,
I re-using the same tooltip element, `createTooltip` is a IIFE and returns a function. tooltip is created once and event listener is also added once.